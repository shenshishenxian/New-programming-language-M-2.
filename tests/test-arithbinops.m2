int main() {
    int i;
    int j;
    int k;
    float f;
    float g;
    float h;
    i = 2;
    j = 4;
    f = 2.2;
    g = 3.3;
    k = i+j;
    printInt(i);
    printInt(j);
    printInt(k);
    printFloat(f);
    printFloat(g);
    printFloat(h);
    printInt(k);
    return 0;
}
