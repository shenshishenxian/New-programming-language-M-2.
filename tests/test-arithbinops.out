2	4	6	2.200000	3.300000	0.000000	6	